worlds/alttp/Bosses.py

```diff
@@ -224,20 +240,20 @@ def can_place_boss(boss: str, dungeon_name: str, level: Optional[str] = None) ->
                                                for boss in boss_table if not boss.startswith("Agahnim"))
 
 
-def place_boss(world, player: int, boss: str, location: str, level: Optional[str]) -> None:
-    if location == 'Ganons Tower' and world.mode[player] == 'inverted':
+def place_boss(multiworld, world, player: int, boss: str, location: str, level: Optional[str]) -> None:
+    if location == 'Ganons Tower' and multiworld.mode[player] == 'inverted':
```

Comment:
(I agree, with el-u's comment. It'd be better to either pass multiworld+player or world, not both.)
Here we could do `world.multiworld.mode` and once options are part of world, this can be simplified.